docsrv/fetcher_test.go

```diff
@@ -12,8 +13,9 @@ const (
 )
 
 func TestReleases(t *testing.T) {
+	apiKey := os.Getenv("GITHUB_API_KEY")
```

Comment:
This is just used for a test (because in travis we used to reach the rate limit since the requests were unauthenticated). This is thought to be run inside the provided container, so the way of configuring the token in real usage is passing it to docker when running it.